pkg/kgo/consumer_group.go

```diff
@@ -2352,8 +2352,43 @@ func (cl *Client) MarkCommitRecords(rs ...*Record) {
 // processing records, you can call this function in a goroutine.
 func (cl *Client) CommitUncommittedOffsets(ctx context.Context) error {
 	// This function is just the tail end of CommitRecords just above.
+	return cl.commitOffsets(ctx, cl.UncommittedOffsets())
+}
+
+// CommitMarkedOffsets issues a synchronous offset commit for any
+// partition that has been consumed from that has marked offsets.
+// Retriable errors are retried up to the configured retry limit, and any
+// unretriable error is returned.
+//
+// This function is useful as a simple way to commit offsets if you have
+// marked offsets with MarkCommitRecords when using AutoCommitMarks.
+// As an alternative if you want to commit specific
+// records, see CommitRecords.
```

Comment:
nit: formatting of this block  seems to have odd wrapping. (I usually just wrap to 80col with vim with `gwap`)

Reply:
indeed

Reply:
this should fixed now